constraint-solver/src/range_constraint.rs

```diff
@@ -14,9 +14,11 @@ use powdr_number::{log2_exact, FieldElement, LargeInt};
 /// Note that the same constraint can have multiple representations.
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Debug, Hash)]
 pub struct RangeConstraint<T: FieldElement> {
-    /// Bit-mask.
+    /// Bit-mask. A value `x` is allowed only if `x & mask == x` (when seen as unsigned integer).
     mask: T::Integer,
     /// Min-max inclusive range. Note that `max` can be smaller than `min`. In this case the range wraps.
+    /// If min <= max (seen as unsigned integers), then the constraint on `x` is `min <= x && x <= max`.
+    /// If min > max, then the constraint is `min <= x || x <= max`.
```

Comment:
I've been meaning to ask what the intuition is here. Could you explain this idea of "wrapped" range?